### DISCUSSION TITLE
how can I jump to match bracket in pseudo C code ?

### DISCUSSION THREAD

**Discussion by baikaishiuc** [Vector35/binaryninja-api#3545] [Q&A] [answered]
I Use '%' key jump between matching parentheses/bracket in IDA

How can I do the same thing in  Binary ninja, have anyone give suggestion to me?

**Comment by psifertex** [comment, marked as answer]
From Pseudo C or HLIL? I can't think of a simple way to do this unfortunately. It might be doable with a UI plugin but again, it's not trivial.

**Reply by baikaishiuc** [reply]
thank you

**Comment by plafosse** [comment]
Please follow this issue if you're interested: https://github.com/Vector35/binaryninja-api/issues/3768
